docker-for-mac/faqs.md

```diff
@@ -154,6 +154,13 @@ response.
 If you do not want auto-send of usage data, use the Stable channel. For more
 information, see [Stable and Edge channels](#stable-and-edge-channels) ("What is the difference between the Stable and Edge versions of Docker for Mac?").
 
+### How does Docker handle Personal Data?
```

Comment:
The subject is a bit broad and should be specific to Docker Desktop. Otherwise it reads like a general statement about Docker, which this is not

Reply:
Thx, all updated with you 4 comments above